docs/architecture.md

```diff
@@ -8,37 +8,47 @@ OSV runs on Google Cloud Platform, with the following main components:
 
 ## Cloud Datastore
 
-All vulnerability data is stored in Cloud Datastore, with the models defined
+All vulnerability data is stored in [Cloud Datastore], with the [models] defined
 [here].
 
+[Cloud Datastore]: https://cloud.google.com/datastore
+[models]:
+https://cloudplatform.googleblog.com/2015/08/Introduction-to-data-models-in-Cloud-Datastore.html
+[here]: https://github.com/google/osv/blob/master/osv/models.py
+
 ## OSS-Fuzz
 
-All of our data is currently sourced from
-[OSS-Fuzz](https://github.com/google/oss-fuzz), and we are working to extend
-this with other sources.
+All of our data is currently sourced from [OSS-Fuzz], and we are working to
```

Comment:
This is actually outdated. We source our data from many other places now. 

Can you instead just say this is an automated fuzzing service where we source a lot of our data and link to http://github.com/google/oss-fuzz ?

Reply:
How about I move this section to the top, change it to Sources and reword to say something along the line of the sources we currently draw from?

Reply:
I've done something along these lines in c3b4781, PTAL.